### PR TITLE
allow disable tabs inkBar animation

### PR DESCRIPTION
**Pull request by xieguanglei**
fix issue [5089](https://github.com/ant-design/ant-design/issues/5089)

### CODE REVIEW

**Comment by xieguanglei** [comment]
not using rebase and .. I closed [last PR](https://github.com/ant-design/ant-design/pull/5446) and pull a new one

File: components/tabs/index.en-US.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Ant Design has 3 types Tabs for different situation.
 | tabPosition  | Position of tabs. Options: top, right, bottom & left | string | top |
 | onEdit       | Callback when tab is added or removed, which is executing when set type as editable-card | (targetKey, action): void | - |
 | hideAdd      | Hide plus icon or not, which is effective when set type as editable-card | boolean | false |
-| animated | Whether to change tabs with animation, this property only works with `tabPosition=top|bottom` | boolean | true |
+| animated | Whether to change tabs with animation, this property only works with `tabPosition=top|bottom`. An {inkBar:boolean, tabPane:boolean} object is also allowed | boolean | true |
```
Comment (benjycui) [inline components/tabs/index.en-US.md]: | animated | Whether to change tabs with animation, this property only works with `tabPosition=top|bottom`.  | boolean \\| Object {inkBar:boolean, tabPane:boolean} | true |

File: components/tabs/index.tsx [hunk 2]
```diff
@@ -153,6 +160,7 @@ export default class Tabs extends React.Component<TabsProps, any> {
 
     const renderTabBar = () => (
       <ScrollableInkTabBar
+        inkBarAnimated={inkBarAnimated}
```
Comment (benjycui) [inline components/tabs/index.tsx]: I cannot find documentation about `ScrollableInkTabBar[inkBarAnimated]`, could you show me the link?

File: components/tabs/index.en-US.md (hunk 1 above)
Comment (xieguanglei) [inline components/tabs/index.en-US.md]: nice, but would this be better ?

| animated | Whether to change tabs with animation, this property only works with tabPosition=top|bottom. | boolean \| {inkBar:boolean, tabPane:boolean} | true |

File: components/tabs/index.tsx (hunk 2 above)
Comment (xieguanglei) [inline components/tabs/index.tsx]: the [rc/tabs documentation](https://github.com/react-component/tabs#libscrollableinktabbar) does not mention about `inkBarAnimated` props, but i think the documentation misses it(may be I can bring up another pr to rc/tabs ? @yiminghe ). 

I read the [source code](https://github.com/react-component/tabs/blob/master/src/InkTabBarMixin.jsx#L89), and I think it's a stable feature.

**Comment by afc163** [comment]
It would be nice to make your PR title readable.

File: components/tabs/index.en-US.md (hunk 1 above)
Comment (benjycui) [inline components/tabs/index.en-US.md]: OK

File: components/tabs/index.tsx (hunk 2 above)
Comment (benjycui) [inline components/tabs/index.tsx]: I try to set `inkBarAnimated={false}`, but the animation is still exist..

Comment (xieguanglei) [inline components/tabs/index.tsx]: right, you need to change index.less, see file changes.

**Comment by benjycui** [comment]
👍

**Pull request merged**